### DISCUSSION TITLE
Version 11 compatibility?

### DISCUSSION THREAD

**Discussion by neostalker** [Muttley/foundryvtt-shadowdark#551] [Q&A]
Is the latest version compatible with Foundry VTT 11? I keep getting the "Error: Unable to load valid system minifest data from "https://gihub.com/Muttley/foundryvtt-shadowdark" Error parsing system manifest data from https:///github.com/Muttley/foundryvtt-shadowdark: Unexpected token '<'," 

![image](https://github.com/Muttley/foundryvtt-shadowdark/assets/24352419/3b071f0e-f63e-4569-af7e-b3c17cf476fd)

Ok, I found the actual Manifest.json but I'm getting a timed out message now.

![image](https://github.com/Muttley/foundryvtt-shadowdark/assets/24352419/361e80e8-410d-405e-ab1f-a82a49de7f86)

**Comment by Muttley** [comment]
The system has been compatable with Foundry v11 since v11 released (before in fact, as it was tested with the development versions of v11).

From those error messages it looks like you are having network issues contacting github from your Foundry install.  What do you get if you paste that URL in a normal browser window?  It should download the system.json file successfully.

**Reply by neostalker** [reply]
Hi Muttley,

Thanks for the quick reply! So when I paste it into Firefox (Updated version) it returns the actual code for the json on the page.

![image](https://github.com/Muttley/foundryvtt-shadowdark/assets/24352419/113e9fa8-685e-4ba9-9964-e374a5dcefc7)

**Reply by Muttley** [reply]
Is your Foundry running remotely on something like Forge, or just locally?  Can you install other systems?  Also, what version of Foundry are you running?

This is very odd.  I've just uninstalled and reinstalled the system here to test, and it worked without any issues.

**Reply by neostalker** [reply]
It's running on my server locally on my network. I'm RDP'd into it. Other systems have installed without issue like shadowrun 5e and this is a fresh install(updated v11 b313) of Foundry.

**Comment by Muttley** [comment]
Did you get that URL in a browser locally, or from the server?  What happens if you put that URL into a browser running on the server?

Other than that, I'm stumped as to what it could be.  We've had 100s of people install the system without issue since 1.4.9 was released a few days ago.  It might be worth asking on the official Foundry discord to see if this is an issue other people have seen.

**Reply by neostalker** [reply]
I got that message from my server. I also received it when I put the link on my browser on my main PC. Could I pull the folder from my main pc and put it on the server and update from there? Do you think that would work. I'll check on Foundry discord as well. I really appreciate your help! I'll check if any other systems are having issues installing as well to narrow things down.

**Reply by Muttley** [reply]
You should be able to just copy the shadowdark system folder, and it will be picked up when you restart Foundry (I'm assuming you've already tried restarting Foundry?)

One other thing to do is hit F12 in Foundry before install the system to bring up the debug console.  Just to see if there are any other messages that may give a clue as to what's happening.

**Reply by neostalker** [reply]
I restarted my computer a few times and lo' and behold it finally installed both Shadowdark and Cyberpunk! Thanks for all your help and creating this DOPE VTT module!

**Reply by Muttley** [reply]
Glad you got it working.
